src/fides/api/ctl/migrations/versions/7e218e880eaf_remove_user_reset_password_scope.py

```diff
@@ -0,0 +1,28 @@
+"""Remove user:reset-password scope
+
+Revision ID: 7e218e880eaf
+Revises: d6c6c6555c86
+Create Date: 2023-01-25 19:16:19.950294
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "7e218e880eaf"
+down_revision = "d6c6c6555c86"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute(
+        "UPDATE fidesuserpermissions SET scopes = array_remove(scopes, 'user:reset-password')"
```

Comment:
@allisonking and I spoke on a call about when `array_remove(...)` was introduced to Postgres — it was introduced in version 11. Fides has only ever shipped with Postgres 12 and beyond so using this function shouldn't be an issue.